vite.md

```diff
@@ -561,6 +564,75 @@ Once a macro has been defined, it can be invoked within your templates. For exam
 <img src="{{ Vite::image('logo.png') }}" alt="Laravel Logo">
 ```
 
+<a name="asset-prefetching"></a>
+## Asset Prefetching
+
+When building a SPA using Vite's code splitting feature, required assets are fetched on each page navigation. This can lead to delayed UI rendering. If this is a problem for your front-end framework of choice, Laravel offers the ability to eagerly prefetch your application's JavaScript and CSS assets on initial page load.
```

Comment:
`When building a SPA` => `When building an SPA`

Reply:
I already treat SPA as an acronym rather than an initialism. Seems the docs has landed on the initialism. Thanks for the review!

Fun poll on this topic: https://x.com/markdalgleish/status/1717709896962158636